tests/cisco_asa/show_arp/cisco_asa_show_arp.raw

```diff
@@ -0,0 +1,5 @@
+        outside 1.2.3.4 444e.6df8.b97e 15
```

Comment:
If the other output for the time stamping is for the ASA, can you add another file with what that looks like to make sure we're able to capture the correct data with the two different formats?